### ISSUE TITLE
No window gets focused under certain conditions when in Sloppy mode

### ISSUE THREAD

**Issue by alteredoxide** [leftwm/leftwm#1145]
### Describe the bug

This seems to be related to an already closed bug: #564 -- When using `Sloppy` behavior with `sloppy_mouse_follows_focus = true`, it seems that any time you switch to different tags while the pointer/cursor is not positioned over a window on the destination tag, no window will gain focus, and using keybindings to focus different windows will not work until you put the cursor over a window.

### Expected behavior (optional)

When windows are moved to new tags, or I switch to a different tag, there will be a window on the currently active tag that gains focus, unless of course there are no windows on the currently active tag.

### Steps to reproduce (optional)

0. Ensure LeftWM is configured with `focus_behavior: Sloppy` and `sloppy_mouse_follow_focus: true`.
1. Start on any tag (doesn't matter if a window is open), and have a second tag not visible that has at least one window open.
2. Move your mouse pointer so that it is positioned over the menu bar or on the edge of the screen, or anywhere such that the pointer will not be positioned over a window when you switch to the second tag.
3. Switch to your second tag.
4. Notice that no window on the destination tag is focused, and you cannot use keybindings to focus different windows.

### OS / Distro

Linux 6.4.12-arch1-1

### LeftWM Check

```text
:: LeftWM version: 0.4.2
:: LeftWM git hash: 357aea5
:: Enabled features:  journald-log lefthk
:: Checking feature dependencies . . .
    -> journald-log OK
    -> lefthk OK
:: Checking for leftwm binaries . . .
    -> Binaries OK 
:: Loading configuration . . .
    -> Configuration loaded OK 
:: Checking keybinds . . .
    -> All keybinds OK
:: Checking environment . . .
    -> Environment OK 
:: Checking theme . . .
    -> Theme OK
```

**Comment by f0ff886f** [comment]
I think this is similar in spirit to https://github.com/leftwm/leftwm/issues/1108. The focus behaviour follows the mouse a little bit too well :D

**Comment by alteredoxide** [comment]
@f0ff886f it does seem like that issue is resulting from the same bug. Thanks for referencing it here.

**Comment by mautamu** [comment]
Resolved in #1285; @alteredoxide feel free to re-open if issue is still not working.
